Vagrantfile

```diff
@@ -27,21 +47,21 @@ if File.exists? VM_SETTINGS_FILE
 else
 
   # Project settings
-  VM_PROJECT = ENV['VM_PROJECT'] || 'suite'                                # Project name
+  VM_PROJECT = ENV['VM_PROJECT'] || 'suite-nonsplit'                                # Project name
```

Comment:
This should be reverted, as the primary use case is still "project perspective".